src/Carbon/Traits/Date.php

```diff
@@ -2630,13 +2630,13 @@ public function __call($method, $parameters)
                 case 'Utc':
                 case 'UTC':
                     return $this->utc;
-                // @call is Check if the current instance has non-UTC timezone.
+                    // @call is Check if the current instance has non-UTC timezone.
```

Comment:
```suggestion
                // @call is Check if the current instance has non-UTC timezone.
```

Reply:
CS gives an error when it is as your suggested.

Reply:
Fixed.

https://github.com/CarbonPHP/carbon/pull/13/commits/dc27804306d1197bc5f8f46f298acf13fa35ed7b